src/components/gas/GasSpeedButton.js

```diff
@@ -80,7 +89,7 @@ const GasSpeedButton = ({ type }) => {
 
   const formatAnimatedGasPrice = useCallback(
     animatedPrice => {
-      const formattedPrice = parseFloat(animatedPrice).toFixed(3);
+      const formattedPrice = parseFloat(Number(animatedPrice).toFixed(3));
```

Comment:
I don't think you need the parseFloat here if we're rendering as a string below.

Reply:
@brunobar79

i actually do need it! i’ll leave comment explaining in code.

but basically, the parseFloat is the thing thats stripping trailing zeros https://linear.app/rainbow/issue/RAI-28/remove-trailing-zeros-from-gas-fees

if `animatedNumber` = `1.010`

then `Number(animatedNumber).toFixed(3)` will return `1.010`

but when wrapped with `parseFloat` it will return `1.01`

Reply:
also it’s important that i moved the `.toFixed` to be *inside* the `parseFloat`